feature-account-impl/src/main/java/jp/co/soramitsu/feature_account_impl/presentation/account/exportaccounts/AccountsForExportViewModel.kt

```diff
@@ -40,15 +41,16 @@ class AccountsForExportViewModel(
 
     private val metaAccount = async(Dispatchers.Default) { interactor.getMetaAccount(payload.metaId) }
 
-    val chainAccountProjections = flowOf { interactor.getChainProjections(metaAccount()) }
+    val chainAccountProjections = flowOf { metaAccount() }
```

Comment:
I see there is a second time when you call metaAccount() and pass it to the interactor. Maybe it would be better to get the metaAccount inside the getChainProjectionsFlow() function?

Reply:
metaAccount should be called from suspend fun, which makes it difficult to use inside getChainProjectionsFlow()